spec/features/courses/index_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+feature 'Index courses', type: :feature do
+  before do
+    stub_omniauth
+    stub_session_create
+    stub_api_v2_request('/api/v2/providers', create(:course))
+  end
+
+  scenario 'viewing the courses index page' do
+    visit '/organisations/AO/courses'
+
+    expect(find('h1')).to have_content('Courses')
```

Comment:
Can we test that the course actually gets listed?

Reply:
The test is incomplete, will update it...

Reply:
Done!